docs/docs/dev_docs/contracts/deploying.md

```diff
@@ -27,11 +27,19 @@ aztec-cli deploy /path/to/contract/artifact.json
 
 Pre-requisite - Compile the contract and generate a type-safe typescript class for it.
 
+Compile the contract:
+
+```bash
+aztec-nargo compile
+```
+
+Generate the ABI and typescript class:
+
 ```bash
-aztec-cli compile /path/to/contract -o target/ -ts target/
+aztec-cli generate-typescript ./aztec-nargo/output/target/path -o src/artifacts
```

Comment:
you might want to rebase master. Charlie's PR of updating docs to use `codegen` has been merged